src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -506,11 +506,11 @@ public void Suspend()
         }
 
         /// <summary>
-        /// TBD
+        /// Handles a <see cref="ISystemMessage"/>
         /// </summary>
         /// <remarks>➡➡➡ NEVER SEND THE SAME SYSTEM MESSAGE OBJECT TO TWO ACTORS ⬅⬅⬅</remarks>
-        /// <param name="systemMessage">TBD</param>
-        public void SendSystemMessage(ISystemMessage systemMessage)
+        /// <param name="systemMessage">The system message to process.</param>
+        public virtual void SendSystemMessage(ISystemMessage systemMessage)
```

Comment:
Working on adding the ability to properly trace `async` / `await` flows within an actor using [Phobos](http://phobos.petabridge.com/) but as it turns out, it's very difficult to instrument that right now since all internal `await` operations flow 100% through the `ActorCell` and don't touch any of the `SendSystemMessage` methods on the `IActorRef` implementations - hence why I've made this `virtual`.

System messages are infrequently sent so the overhead of this change (virtual method dispatch overhead) should be trivial.